Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php

```diff
@@ -37,50 +37,37 @@
 final readonly class Node
```

Comment:
the api annotation above

>  * @api Note: The constructor is not part of the public API

need to be adjsuted as well

Reply:
Why? `create` is a constructor, too. Or how would you rephrase it?

Reply:
ah okay then its fine ^^